src/nvim/buffer.c

```diff
@@ -1844,7 +1844,7 @@ buf_T *buflist_new(char_u *ffname_arg, char_u *sfname_arg, linenr_T lnum,
       EMSG(_("W14: Warning: List of file names overflow"));
       if (emsg_silent == 0) {
         ui_flush();
-        os_delay(3000L, true);  // make sure it is noticed
+        os_delay(3001L, true);  // make sure it is noticed
```

Comment:
What is the rationale between these magic numbers and does it even apply to us?

Reply:
:man_shrugging: It's a vim patch. I omitted the change where `ui_delay` is logged. I don't know the alternative to Vim's `ch_log()`.

On Vim, `ui_delay()` runs either `gui_wait_for_chars()` or `mch_delay` (which runs `WaitForChar()`) so the extra milliseconds matter.

Reply:
Updated the patch to use `DLOG` instead of  `ch_log`.